include/modem/lte_lc.h

```diff
@@ -504,6 +504,21 @@ enum lte_lc_modem_evt {
 
 	/** The device is overheated and the modem is therefore deactivated. */
 	LTE_LC_MODEM_EVT_OVERHEATED,
+
+	/** The modem does not have an IMEI */
+	LTE_LC_MODEM_EVT_NO_IMEI,
+
+	/** Selected CE-level in RACH procedure is 0, see 3GPP TS 36.331 for details. */
+	LTE_LC_MODEM_EVT_CE_LEVEL_0,
+
+	/** Selected CE-level in RACH procedure is 1, see 3GPP TS 36.331 for details. */
+	LTE_LC_MODEM_EVT_CE_LEVEL_1,
+
+	/** Selected CE-level in RACH procedure is 2, see 3GPP TS 36.331 for details. */
+	LTE_LC_MODEM_EVT_CE_LEVEL_2,
+
+	/** Selected CE-level in RACH procedure is 3, see 3GPP TS 36.331 for details. */
+	LTE_LC_MODEM_EVT_CE_LEVEL_3,
```

Comment:
Should we say that these are only supported by modem firmware version >= 2.0.0?

Reply:
Added